src/lib/viewers/controls/annotations/AnnotationsControls.tsx

```diff
@@ -6,27 +6,33 @@ import IconDrawing24 from '../icons/IconDrawing24';
 import IconHighlightText16 from '../icons/IconHighlightText16';
 import IconRegion24 from '../icons/IconRegion24';
 import useFullscreen from '../hooks/useFullscreen';
-import { AnnotationMode } from '../../../types';
+import { AnnotationMode, TargetingApi } from '../../../types';
 import './AnnotationsControls.scss';
 
 export type Props = {
     annotationColor?: string;
     annotationMode?: AnnotationMode;
+    experiences: {
+        [name: string]: TargetingApi;
+    };
     hasDrawing?: boolean;
     hasHighlight?: boolean;
     hasRegion?: boolean;
     onAnnotationModeClick?: ({ mode }: { mode: AnnotationMode }) => void;
     onAnnotationModeEscape?: () => void;
+    setWasClosedByUser?: (experienceName: string | undefined) => void;
 };
 
 export default function AnnotationsControls({
     annotationColor = bdlBoxBlue,
     annotationMode = AnnotationMode.NONE,
+    experiences,
```

Comment:
Can we avoid prop drilling by using context?

Reply:
Yes, I think that's a good idea!